client/src/components/Shared/Form/form.style.js

```diff
@@ -0,0 +1,118 @@
+import styled from 'styled-components';
+
+export const StyledForm = styled.form`
+display:flex;
+justify-content: center;
+flex-direction:column;
+`;
+export const StyledImgCon = styled.div`
+display:flex;
+width: 180px;
+height:180px;
+background-color: #FBFBFB;
+border-radius:50%;
+margin:auto;
+justify-content: center;
+align-items: center;
+overflow:hidden;
+`;
+
+export const StyledImg = styled.img`
+height: 200px;
+`;
+
+export const StyledInformation = styled.h2`
+margin-left:40px;
+font-size: 18px;
+font-family:'Assistant';
+color:#909090;
+text-align:left;
+@media (max-width: 650px) {
+  font-size: 12px;
+}
+`;
+
+export const StyledNotic = styled.h2`
```

Comment:
What does Notic mean?

Reply:
![Screenshot from 2019-03-24 11-44-58](https://user-images.githubusercontent.com/32957970/54877687-61ec9b80-4e2a-11e9-8a30-a16ba26787c0.png)

Reply:
Please verify the items marked this is notic

Reply:
Do you mean 'notice'?